File: src/frontend/test/src/domain/entities/TypeDestination.ts
export interface ITypeDestination {
    id?: number;
    nomTypeDestination: string;
    descriptionTypeDestination?: string;
}

export class TypeDestination implements ITypeDestination {
    constructor(
        public nomTypeDestination: string,
        public descriptionTypeDestination?: string,
        public id?: number
    ) {}
}

File: src/backend/NHP_Domain/Entities/TypeDestination.cs
namespace NHP_Domain.Entities;

public class TypeDestination
{
    public long Id { get; set; }
    public String NomTypeDestination { get; set; } = String.Empty;
    public String? DescriptionTypeDestination { get; set; }
    
    // OneToMany
    //public virtual List<Logement>? Logements { get; set; } = new();

    public override string ToString()
    {
        return $"typeDestination: {NomTypeDestination} " +
               $"descriptionTypeDestination: {DescriptionTypeDestination}";
    }
}